sources/index.js

```diff
@@ -82,11 +99,11 @@ class ShowMore {
       const originalText = element.innerHTML.trim();
       let elementText = element.textContent.trim();
 
-      const orgTexReg = originalText
-        .replace(this.regex.br, '')
-        .replace(this.regex.newLine, '')
-        .replace(this.regex.space, ' ')
-        .replace(this.regex.html, '')
+      let orgTexReg = originalText
+      if(this.regex.br && this.regex.br.match) orgTexReg = orgTexReg.replace(this.regex.br.match, this.regex.br.replace)
```

Comment:
Perhaps a better solution would be a for...in loop
```
for (let key in this.regex) {
  const {match, replace} = this.regex [key];
  if (key && match) orgTexReg = orgTexReg.replace (match, replace);
}
```

Reply:
Right after the pull request I had the same idea. Have updated the code.